{"name": "test_login_with_valid_credentials[aziguy_three@gmail.com-secure_password_three]", "status": "passed", "parameters": [{"name": "email_address", "value": "'aziguy_three@gmail.com'"}, {"name": "password", "value": "'secure_password_three'"}], "start": 1722274050910, "stop": 1722274053208, "uuid": "2f1dbdde-ac6f-4210-a54e-1e5c8cd631f1", "historyId": "b995b8c543151aaa4463ff4321b655ec", "testCaseId": "dc8ec9f4f04bfae47cc257d2b40052bd", "fullName": "tests.test_loging.TestLoging#test_login_with_valid_credentials", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_teardown', 'log_on_failure')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_loging"}, {"name": "subSuite", "value": "TestLoging"}, {"name": "host", "value": "kengni"}, {"name": "thread", "value": "3344098-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_loging"}]}